gcp_variant_transforms/testing/temp_dir.py

```diff
@@ -40,20 +44,38 @@ def get_path(self):
     """Returns the path to the temporary directory."""
     return self._tempdir
 
-  def create_temp_file(self, suffix='', lines=None):
+  def create_temp_file(
+      self, suffix='', lines=None,
+      compression_type=filesystem.CompressionTypes.UNCOMPRESSED):
     """Creates a temporary file in the temporary directory.
+
     Args:
       suffix (str): The filename suffix of the temporary file (e.g. '.txt')
       lines (List[str]): A list of lines that will be written to the temporary
         file.
+      compression_type (``CompressionTypes``): Enum specifying compression type.
```

Comment:
This is actually labeled as a (str) in other classes. I think this labeling it as an enum is probably more appropriate to not break the abstraction, but I'm not sure if we should favor that over consistency.

Reply:
ah, I didn't realize it's a string in other classes. Changes here to be str as well.